frontend/model/contracts/group.js

```diff
@@ -536,13 +540,16 @@ sbp('chelonia/defineContract', {
           })
         })
       }),
-      process ({ data, meta, contractID }, { state, getters }) {
+      process ({ contractID, data, meta }, { state, getters }) {
         // TODO: checkpointing: https://github.com/okTurtles/group-income/issues/354
         const initialState = merge({
+          chatRooms: {},
+          invites: {},
           payments: {},
           paymentsByPeriod: {},
-          thankYousFrom: {}, // { fromUser1: { toUser1: msg1, toUser2: msg2, ... }, fromUser2: {}, ...  }
-          invites: {},
+          profiles: {
+            [meta.username]: initGroupProfile(meta.identityContractID, meta.createdDate)
+          },
```

Comment:
Similarly, here's a bunch of changes involving re-arranging the existing code that didn't need to be made... why?

Reply:
I tend to find unsorted entries a bit distracting, so I couldn't resist the urge to sort them. The same goes for helper functions, but reordering them would have caused a big git diff unfortunately.

Reply:
Of course I can revert this change for this PR

Reply:
We can leave this for, but, for the future, please send a minimal PR with only essential changes or clear improvements/simplifications